client/my-sites/plans/current-plan/current-plan-thank-you/style.scss

```diff
@@ -1,31 +1,41 @@
 .current-plan-thank-you {
+	display: flex;
+	flex-direction: row;
 	max-width: 720px;
-	text-align: center;
 
 	@include breakpoint( '>800px' ) {
 		font-size: 16px;
+		padding: 20px 0;
+	}
+
+	h1 {
```

Comment:
Let's use `.current-plan-thank-you__title` class instead to target this, so that we don't rely on HTML primitives for styling.

Reply:
👍